### ISSUE TITLE
Extra Line Return w/ 2 tabs added before </script> & </style> tags

### ISSUE THREAD

**Issue by StreamUtilitySuite** [AngleSharp/AngleSharp#1207]
### Prerequisites

- [X] Can you reproduce the problem in a [MWE](https://en.wikipedia.org/wiki/Minimal_working_example)?
- [X] Are you running the latest version of AngleSharp?
- [X] Did you check the FAQs to see if that helps you?
- [X] Are you reporting to the correct repository? (there are multiple AngleSharp libraries, e.g., `AngleSharp.Css` for CSS support)
- [X] Did you perform a search in the issues?

### Description

Whenever I parse some HTML code and save it to a file, it adds a line return w/ 2 tabs just before the </script> and </style> tags.

What ends up happening is that the more I parse and save, the great that distance becomes.

![ExtraLineReturns](https://github.com/user-attachments/assets/6c760815-4ff6-4a42-8de8-55a024c6de26)

### Steps to Reproduce

**Using this HTML code:**
[SampleHTML.txt](https://github.com/user-attachments/files/17447433/SampleHTML.txt)

**Using this C# code:**

> var parser = new HtmlParser();
var document = parser.ParseDocument(htmlString);
var sw = new StringWriter();
document.ToHtml(sw, new PrettyMarkupFormatter());
string output = sw.ToString();

You can see that the output string has extra line returns in it

### Expected Behavior

Formats HTML properly without extra line returns before the </script> and </style> tags

### Actual Behavior

Formats HTML properly with extra line returns before the </script> and </style> tags

### Possible Solution / Known Workarounds

The only way I've been able to get it to stop is to search line by line and replace "\t\t" with nothing if that's all that's on the line which is a bit tedious

**Comment by FlorianRappl** [comment]
Please provide code. Right now I think you are not providing a stream but read the file beforehand, i.e. having CRLF instead of just LF.

Would be good to see what you are actually doing before having any look at this.

**Comment by StreamUtilitySuite** [comment]
Here is the full section of code. 

I am using HtmlAgilityPack to write out my HTML file from my C# code but for some reason, if the HTML isn't pretty printed, the CSS doesn't seem to work which is a weird issue in and of itself.

The code below is right after I save the file using HtmlAgilityPack so I basically reload it to get the latest, parse all it's text and then format it. I ended up finding a better workaround yesterday where I replace "\n\t\t\n" with just "\n" and that saves time and resources and gives me what I want.

I did check the output of documentChat.Text and the extra line return isn't there until it gets to the document.ToHtml function.

documentChat.Load(Globals.CHAT_WINDOW_PATH);
var parser = new HtmlParser();
var document = parser.ParseDocument(documentChat.Text);
var sw = new StringWriter();
document.ToHtml(sw, new PrettyMarkupFormatter());
File.WriteAllText(Globals.CHAT_WINDOW_PATH, sw.ToString().Replace("\n\t\t\n", "\n"));
